src/components/CreateCards.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+function CreateCards(props) {
+  return (
+    <div className="container">
+      {props.cardInfo.map((data, index) => {
+        return (
```

Comment:
I think that this component should render a single card, so div with `container` class from line 5 and map function should stay in `TouristInfoCards` component. Also React component names should be nouns - I wouldn't use word `create` in the name as it's a verb. And I would stick to arrow functions🙂